src/sales/order-update.md

```diff
@@ -56,3 +77,29 @@ Canceling an order changes its status from `Pending` to `Canceled`.
     _Cancel Order_
 
 The status of the order is now `Canceled`.
+
+### Button Bar for Pending Orders
```

Comment:
If these are both H3, they are under the "Cancel a pending order" H2 (which doesn't really make sense). Maybe move each under the "Pending Orders" and "Processing Orders" H2s. 

Also, the `{: .buttons-table }` does not work unless you add the style declaration to the bottom of the page. Adjust the width of that first column according to what is needed to keep the button items from wrapping in the cell.

Reply:
Fixed. That's a good trick!